radis/test/misc/test_utils.py

```diff
@@ -1,14 +1,14 @@
 from __future__ import print_function, absolute_import, division, unicode_literals
 import pytest
 import os
-from os.path import dirname, basename, join, expanduser
+from os.path import dirname, basename, join, expanduser, abspath
 from os import listdir
 import fnmatch
 from radis.misc.utils import get_files_from_regex
 
 
 # Test to compare basenames of files whose names match the regular expression passed.
-FOLDER_PATH_EXAMPLE = expanduser("~/radis/radis/test/files")
+TEST_FOLDER_PATH = dirname(abspath(dirname(__file__)))
```

Comment:
That works, but you can use getTestFile in case we ever move test_utils.py, or the test files folder.

```
TEST_FILES_PATH = dirname(getTestFile("."))
```
(this returns something like "~radis/radis/test/files/")

Reply:
I think getTestFile will throw an error because, before returning the file path, it's existance is being checked.

Reply:
In the latest version yes, so you cannot use getTestFile("*.par"), but you can use getTestFile(".") which will return the folder name (which exists!)
At least it works on my local Windows version.